### PR TITLE
Update ceph CSI driver details in the drivers list

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: book/src/drivers.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [AWS FSx for Lustre](https://github.com/aws/aws-fsx-csi-driver) | `fsx.csi.aws.com` | v0.3 | A Container Storage Interface (CSI) Driver for AWS FSx for Lustre (EBS) | Persistent | Read/Write Multiple Pods | No | No | No | ? | ?
 [Azure disk](https://github.com/andyzhangx/azuredisk-csi-driver) | `disk.csi.azure.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Azure disk | Persistent | Read/Write Single Pod | Yes | No | No | ? | ?
 [Azure file](https://github.com/andyzhangx/azurefile-csi-driver) | `file.csi.azure.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for Azure file | Persistent | Read/Write Multiple Pods | Yes | No | No | ? | ?
-[CephFS](https://github.com/ceph/ceph-csi) | `cephfs.csi.ceph.com` | v0.3, v1.0 | A Container Storage Interface (CSI) Driver for CephFS | Persistent | Read/Write Multiple Pods | Yes | No | No | ? | ?
```
Comment (msau42) [inline book/src/drivers.md]: can you also provider yes/no for the question marks?

Comment (humblec) [reply book/src/drivers.md]: Sure @msau42 , followed the pattern of other drivers :)

Comment (humblec) [reply book/src/drivers.md]: @msau42 Done. PTAL . Thanks!

**Pull request merged**